akka-actor/src/main/scala/akka/pattern/CircuitBreaker.scala

```diff
@@ -317,6 +318,96 @@ class CircuitBreaker(
   }
 
   /**
+   * Adds a callback to execute when call finished with success.
+   *
+   * The callback is run in the [[scala.concurrent.ExecutionContext]] supplied in the constructor.
+   *
+   * @param callback Handler to be invoked on successful call, where passed value is elapsed time in nanoseconds.
+   * @return CircuitBreaker for fluent usage
+   */
+  def onCallSuccess(callback: Long ⇒ Unit): CircuitBreaker = onCallSuccess(new Consumer[Long] {
+    def accept(result: Long): Unit = callback(result)
+  })
+
+  /**
+   * JavaAPI for onCallSuccess
+   *
+   * @param callback Handler to be invoked on successful call, where passed value is elapsed time in nanoseconds.
+   * @return CircuitBreaker for fluent usage
+   */
+  def onCallSuccess(callback: Consumer[Long]): CircuitBreaker = {
```

Comment:
@maciekciolek They have the same signature as `onCallBreakerOpen` and @bantonsson has verified that it is alright (we were lucky). However, while you are anyway doing this please add new methods with the same naming so that we have a consistent naming. It should look like:

```scala
  /**
   * JavaAPI for onClose
   *
   * @param callback Handler to be invoked on state change
   * @return CircuitBreaker for fluent usage
   */
  def addOnCloseListener(callback: Runnable): CircuitBreaker = {
    Closed addListener callback
    this
  }
  
  /**
   * JavaAPI for onClose
   *
   * @param callback Handler to be invoked on state change
   * @return CircuitBreaker for fluent usage
   */
  @deprecated("Use addOnCloseListener instead", "2.5.0")
  def onClose(callback: Runnable): CircuitBreaker = {
    addOnCloseListener(callback)
  }
```

BTW, looks like we are close to merging this and it would be great to get it in before 2.5.0-RC1 on Friday